<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of managers</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="LoginPageGroupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>621</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display Semib</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#LoginPageGroupBox {
background-image:
 url(:/Farm/images/Farm.jpg);
}</string>
    </property>
    <property name="title">
     <string>LoginPage</string>
    </property>
    <widget class="QPushButton" name="EyeBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Password...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsernameLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Username...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmailLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Email...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PhoneNumberLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="placeholderText">
      <string>Phone Number...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CountryComboBox">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>170</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RefreshBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ContinueBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>350</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>continue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MoneyLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Money amount...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="captchaLineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Captcha...</string>
     </property>
    </widget>
    <widget class="QLabel" name="showCaptcha">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ConfirmBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>confrim</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="NumberOfMaganersSpinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
